src/entities/Checkpoint.ts

```diff
@@ -5,33 +5,76 @@ import { DividendDistribution } from './DividendDistribution';
 import { ShareholderBalance, ErrorCode } from '../types';
 import { PolymathError } from '../PolymathError';
 
+/**
+ * Represents a checkpoint identifier
+ */
 export interface UniqueIdentifiers {
+  /**
+   * st identifier
+   */
   securityTokenId: string;
+  /**
+   * checkpoint id used to query historical balances
```

Comment:
numerical index of the checkpoint. The higher the index, the more recent the checkpoint